### PR TITLE
seperate static file of bootstrap and os repos from api.server into file.server

### PR DESCRIPTION
**Pull request by lanchongyizu**
* seperate static file of bootstrap and os repos from api.server into file.server

@RackHD/corecommitters @iceiilin @cgx027 @panpan0000 @benbp

### CODE REVIEW

**Comment by rackhdmirror** [comment]
@RackHD/corecommitters please review.

**Comment by panpan0000** [comment]
:+0. defer to experts to review :-)

**Comment by panpan0000** [comment]
test this please

**Comment by iceiilin** [comment]
:+1:

File: lib/task.js [hunk 1]
```diff
@@ -109,10 +109,26 @@ function factory(
                 Task.configCache.apiServerPort
             );
         }
+        var fileServerUri;
+        if (_.has(Task.configCache, 'fileServerAddress') &&
+            _.has(Task.configCache, 'fileServerPort') &&
+            _.has(Task.configCache, 'fileServerPath')) {
```
Comment (cgx027) [inline lib/task.js]: I would suggest not making fileServerPath a mandatory field. If user meant to use the root directory of the file server, they would got a chance of not specifying the fileServerPath.

This is a non blocking anyway.

**Pull request merged**